appcues/src/main/java/com/appcues/trait/ContainerDecoratingTrait.kt

```diff
@@ -15,5 +16,5 @@ interface ContainerDecoratingTrait : ExperienceTrait {
     val containerComposeOrder: ContainerDecoratingType
 
     @Composable
-    fun BoxScope.DecorateContainer()
+    fun BoxScope.DecorateContainer(wrapperInsets: PaddingValues)
```

Comment:
added wrapperInsets (safe area) information on ContainerDecoratingTrait

Reply:
Should give a descriptive comment here please, since part of public API